Assets/SolarSystem/OrbitalTrails/OrbitalTrail.mat

```diff
@@ -93,7 +93,7 @@ Material:
       second: 0.04
     - first:
         name: _GlobalScale
-      second: 10.079262
```

Comment:
This change doesn't seem to have anything to do with your PR and is probably a side effect of Unity's editor. There are pending changes in the Phone branch that will help with Unity's editor and material modification. Please remove this file from your PR.